tools/sgen/input/FluentUIStyle.yml

```diff
@@ -403,3 +403,59 @@ MSFPrimaryHeaderTokens extends MSFHeaderFooterTokens:
   textColor: $Colors.Foreground.neutral1
   textFont: $Typography.headline
   topPadding: $Spacing.medium
+
+AP_MSFNotificationTokens:
+  backgroundColor: $Colors.Brand.tint40
+  foregroundColor: "FluentUIColor(light: $Colors.Brand.shade10, dark: $Colors.Brand.shade30)"
+  cornerRadius: $Border.radius.xLarge
+  presentationOffSet: $Spacing.small
+  horizontalPadding: 16pt
+  verticalPadding: 12pt
+  verticalPaddingForOneLine: 16pt
+  horizontalSpacing: 16pt
+  minimumHeight: 64.0
```

Comment:
Is there a reason that some of these values are in `pt` and some are floats? I think the parser treats them the same regardless, but I was curious if there's a semantic meaning. 

(I don't think I've personally been good about any such difference myself, so this is more of a question than a concern.)

Reply:
I used pt for the padding/spacing tokens because I originally used the $Spacing definitions above (which I've gone back to in the latest commit) and those use `pt` while other control tokens use floats for their heights.